python/mlcroissant/mlcroissant/_src/structure_graph/nodes/metadata_test.py

```diff
@@ -42,20 +42,12 @@ def test_from_jsonld(conforms_to: CroissantVersion):
         constants.ML_COMMONS_IS_LIVE_DATASET(ctx): False,
         constants.SCHEMA_ORG_URL: "https://mlcommons.org",
         constants.SCHEMA_ORG_VERSION: "1.0.0",
-        constants.ML_COMMONS_DATA_BIASES(ctx): "data_biases",
```

Comment:
We can remove this as nobody has adopted them yet?

Reply:
If we remove it before the launch, we will avoid confusion with the RAI extension attributes.